### PR TITLE
docs: add intro text to imageryintro.html

### PR DESCRIPTION
**Pull request by neteler**
This PR adds an overview text to `imagery/imageryintro.html` including a link to addons to not start this intro text without context.

(aside, addons URL fix in `g.gui.gmodeler.html`)

### CODE REVIEW

File: imagery/imageryintro.html [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
+aerial photography. Image processing in GRASS GIS supports a wide
+range of preprocessing operations, such as import, georeferencing,
+radiometric calibration and atmospheric correction. It is particularly
+suited for handling Earth observation data, allowing users to analze
+multispectral and temporal datasets. GRASS GIS supports image classification,
+sensor fusion, point cloud statistics and more. From  to the calculation
+of vegetation indices, it enables the analysis of environmental,
+agricultural, and land cover dynamics. Numerous
+<a href="https://grass.osgeo.org/grass8/manuals/addons/#i">addons</a>
+are available as well.
```
Comment (nilason) [inline imagery/imageryintro.html:16]:
```suggestion
GRASS GIS provides a powerful suite of tools for the processing and
analysis of geospatial raster data, including satellite imagery and
aerial photography. Its image processing capabilities encompass a broad
range of preprocessing operations, such as data import, georeferencing,
radiometric calibration and atmospheric correction. It is particularly
suited for handling Earth observation data, enabling the analysis of
multispectral and temporal datasets. GRASS GIS supports advanced
functionalities such as image classification, sensor fusion, and point
cloud statistics. The calculation of vegetation indices further enables
analyses of environmental, agricultural, and land cover dynamics. An
extensive collection of
<a href="https://grass.osgeo.org/grass8/manuals/addons/#i">addons</a>
further enhances its analytical capabilities.
```

File: imagery/imageryintro.html [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+range of preprocessing operations, such as data import, georeferencing,
+radiometric calibration and atmospheric correction. It is particularly
+suited for handling Earth observation data, enabling the analysis of
+multispectral and temporal datasets. GRASS GIS supports advanced
+functionalities such as image classification, sensor fusion, and point
+cloud statistics. The calculation of vegetation indices further enables
+analyses of environmental, agricultural, and land cover dynamics. An
+extensive collection of
+<a href="https://grass.osgeo.org/grass8/manuals/addons/#i">addons</a>
+further enhances its analytical capabilities.
```
Comment (wenzeslaus) [inline imagery/imageryintro.html:17]: Given you link to imagery:

```suggestion
further enhances its image processing capabilities.
```

**Review by wenzeslaus — CHANGES_REQUESTED** [review]
I also noticed that the words remote sensing are not mentioned. Too many terms to cover: EO, RS, image processing, imagery. I'm not sure what to suggest.

File: imagery/imageryintro.html (hunk 1 above)
Comment (neteler) [reply imagery/imageryintro.html]: Thanks @nilason!

**Review by nilason — APPROVED** [review]
Great!

**Comment by neteler** [comment]
> Too many terms to cover: EO, RS, image processing, imagery.

Attempt added in f578e6d.

**Review by wenzeslaus — APPROVED** [review]
Thanks!

**Pull request merged**